sdk-network-providers/transaction_status.md

```diff
@@ -0,0 +1,21 @@
+class TransactionStatus:
+    status: string
+
+    // The constructor is not captured by the specs; it's up to the implementing library to define it.
+    // can be something like:
+    constructor(status: string | None);
+
+    is_pending(): boolean;
```

Comment:
The proxy is now able to provide the following transaction statuses: 
"pending", "success", "fail", "invalid"
From what you have seen already, is it better to have all these is_xxx functions?